modules/microshift-config-yaml.adoc

```diff
@@ -44,21 +49,22 @@ debugging:
 <8> Subject Alternative Names for API server certificates.
 <9> Log verbosity. Valid values for this field are `Normal`, `Debug`, `Trace`, or `TraceAll`.
 
+[id="microshift-yaml-custom_{context}"]
+== Custom settings
+To create custom configurations, you must create a `config.yaml` file in the `/etc/microshift/` directory, then change any settings that should override the defaults before starting or restarting {product-title}.
+
 [IMPORTANT]
 ====
-Restart {product-title} after changing any configuration settings to have them take effect. {product-title} reads the configuration file only on start.
+Restart {product-title} after changing any configuration settings to have them take effect. The `config.yaml` file is read only when {product-title} starts.
 ====
 
-[id="microshift-yaml-custom_{context}"]
-== Custom settings
-You must change any settings that should override the defaults before starting {product-title}.
-
-advertiseAddress::
+[id="microshift-yaml-advertiseAddress_{context}"]
```

Comment:
Not blocking merge for this, but `id`s should be all lowercase.

Reply:
oh yeah, just copied it, thx, will bolo